actors/builtin/power/power_test.go

```diff
@@ -428,6 +425,7 @@ func TestPowerAndPledgeAccounting(t *testing.T) {
 	})
 
 	t.Run("threshold only depends on qa power, not raw byte", func(t *testing.T) {
+		t.Skip("switching to consensus min based on raw byte power for space race")
```

Comment:
Please adjust this and the tests below to instead verify that the raw byte power is consulted instead of the QA power.